lib/tls/session.toit

```diff
@@ -697,7 +692,7 @@ class ToitHandshake_:
     handshake-hasher /checksum.Checksum := cipher-suite_.hmac-hasher.call
     hello := client-hello-packet_
     handshake-hasher.add hello[5..]
-    sent := session_.writer_.write hello
+    sent := session_.writer_.try-write hello
```

Comment:
I think this should be write, not try-write, and the assert is no longer needed?